src/components/StakingVaults/EarnOpportunities.tsx

```diff
@@ -36,14 +36,15 @@ export const EarnOpportunities = ({ assetId }: EarnOpportunitiesProps) => {
     dispatch,
   } = useWallet()
   const asset = useAppSelector(state => selectAssetById(state, assetId))
-  const vaults = useYearnVaults()
+  const { vaults } = useVaultBalances()
   const { data: foxyBalancesData } = useFoxyBalances()
+
   const { onlyVisibleFoxFarmingOpportunities, foxEthLpOpportunity } = useFoxEth()
   const featureFlags = useAppSelector(selectFeatureFlags)
   //@TODO: This needs to be updated to account for accountId -- show only vaults that are on that account
 
   const allRows = useNormalizeOpportunities({
-    vaultArray: vaults,
+    vaultArray: Object.values(vaults),
```

Comment:
Note: This is fine for now, but we should either be able to query vaults by provider in the future, instead of needing to `Object.values()` them, or expose their value as well